src/bump/type.go

```diff
@@ -72,3 +81,19 @@ func Bump(version *semver.Version, bt Type) *semver.Version {
 
 	return &v
 }
+
+// NameToType returns the bump type from a string. The string should be from a constant constant of bump.Name
+// or it will return bump.None.
+func NameToType(name string) Type {
```

Comment:
I confess this one is opinionated
```suggestion
func FromName(name string) Type {
```

Reply:
I am letting more people review the PR so we can have more opinions on this.

Reply:
If the package name were `type` I would definitely prefer `FromName` because it would be used as `type.FromName`.  As it is in `bumper` package I think having the type in the function name is a bit clearer, another option would be `bumper.TypeFromName`.